spring-cloud-gcp-pubsub/pom.xml

```diff
@@ -42,5 +42,34 @@
 			<artifactId>spring-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<!-- SI channel adapters -->
+		<dependency>
+			<groupId>org.springframework.integration</groupId>
+			<artifactId>spring-integration-core</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>com.google.api</groupId>
+			<artifactId>api-common</artifactId>
+			<!-- This shouldn't be required here. -->
+      <version>1.0.0</version>
+    </dependency>
 	</dependencies>
+
+	<!-- Must also get rid of this -->
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.codehaus.mojo</groupId>
+				<artifactId>animal-sniffer-maven-plugin</artifactId>
```

Comment:
This may be due usage of jdk8 as requirement on the pom. You need to add an empty .jdk8 file to the project so our build tools ignore jdk7 enforcement